docs/maintainer-guide/releases.md

```diff
@@ -1,53 +1,47 @@
-6.  If a new version of any `stylelint-config-*` is required, repeat step 5 for that repo.
-7.  Update the online demo by changing to the `stylelint-demo` repo:
-    1.  Run `npm install -S stylelint@latest`
-    2.  Run `npm test`
-    3.  Commit and _push up_ these changes.
-    4.  Go to [https://stylelint.io/demo](https://stylelint.io/demo) and confirm the update was automatically deployed.
-8.  Update the website documentation by changing to the `stylelint.io` repo:
-    1.  Run `npm install -D stylelint@latest`
-    2.  Run `npm test`
-    3.  Commit and _push up_ these changes.
-    4.  Go to [https://stylelint.io](https://stylelint.io) and confirm the update was deployed correctly. (It takes some time for Netlify to publish)
-9.  Compose a tweet that announces the release, communicates what has changed and links to the appropriate heading in the CHANGELOG on [https://stylelint.io](https://stylelint.io).
-
-## New Release Issue Template
+    3.  Commit these changes.
+    4.  Push these changes.
+    5.  Confirm the changes are correct at [https://github.com/stylelint/stylelint](https://github.com/stylelint/stylelint).
+    6.  Run `npm run release`.
+    7.  Select the version that matches the one from the changelog.
+    8.  Copy and paste the changelog entries for the published version from [changelog](../../CHANGELOG.md) when the GitHub release page opens.
+    9.  Confirm the publishing of the package to [https://www.npmjs.com/package/stylelint](https://www.npmjs.com/package/stylelint).
+    10. Confirm the creation of the release at [https://github.com/stylelint/stylelint/releases](https://github.com/stylelint/stylelint/releases).
+6.  If a new version of any `stylelint-config-*` is required, repeat step 5 for that repository.
+7.  Update the online demo by changing to the `stylelint-demo` repository:
+    1.  Run `npm install -S stylelint@latest`.
+    2.  Run `npm test`.
+    3.  Commit these changes.
+    4.  Push these changes.
+    5.  Confirm the deployment of the update to [https://stylelint.io/demo](https://stylelint.io/demo).
+8.  Update the website documentation by changing to the `stylelint.io` repository:
+    1.  Run `npm install -D stylelint@latest`.
+    2.  Run `npm test`.
+    3.  Commit these changes.
+    4.  Push these changes.
+    5.  Confirm the deployment of the update to [https://stylelint.io](https://stylelint.io)
+9.  Compose a tweet that:
+    -   announces the release
+    -   communicates what has changed
+    -   links to the appropriate heading in the changelog on [https://stylelint.io](https://stylelint.io).
+
+## New release issue template
 
 ```markdown
 - [ ] stylelint release
 - [ ] stylelint-config-recommended update/release
 - [ ] stylelint-config-standard update/release
 - [ ] stylelint-demo update
 - [ ] stylelint.io update
+- [ ] vscode-stylelint update
```

Comment:
[question] Is there a detail descrtiption about "vscode-stylelint update"??? 🤔

Reply:
Not yet. @ntwb said that he would add it when he next has time. It'll likely be in a follow-up pull request once this one merged.

Reply:
I see 👍